src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -211,6 +212,8 @@ private static bool HasRetryAfterHeader(HttpResponse response)
                 requestMessage.Method = method;
                 requestMessage.Content = body;
 
+                logger.Verbose($"Sending request from HttpManager. Method: {method}. URI path: {endpoint.Scheme}://{endpoint.Authority}{endpoint.AbsolutePath}. ");
```

Comment:
Do you want to log the HTTP status of the response as well?

Reply:
Done. We also have additional logging for response errors in the outer method.